4-storage/relational-database-shredder/project/BuildSettings.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2012-2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and
+ * limitations there under.
+ */
+import sbt._
+import Keys._
+
+object BuildSettings {
+
+  lazy val buildSettings = Seq[Setting[_]](
+    organization  := "com.snowplowanalytics",
+    scalaVersion  := "2.10.4",
+    scalacOptions := compilerOptions,
+    parallelExecution in Test := false, // Parallel tests cause havoc with Spark
```

Comment:
Yup, I've had issues with this in the past as in [spark-kafka-writer](https://github.com/benfradet/spark-kafka-writer) for example.
Though, tbh, I haven't tried with those particular tests.